terminal/TerminalClient.cpp

```diff
@@ -135,6 +136,29 @@ int main(int argc, char** argv) {
   FLAGS_logbuflevel = google::GLOG_INFO;
   srand(1);
 
+  Options options = {
+    NULL, //username
+    NULL, //host
+    NULL, //sshdir
+    NULL, //knownhosts
+    NULL, //ProxyCommand
+    0, //timeout
+    0, //port
+    0, //StrictHostKeyChecking
+    0, //ssh2
+    0, //ssh1
+    NULL, //gss_server_identity
+    NULL, //gss_client_identity
+    0 //gss_delegate_creds
+  };
+  char* home_dir = ssh_get_user_home_dir();
+  ssh_options_set(&options, SSH_OPTIONS_HOST, FLAGS_host.c_str());
+  /* First parse user-specific ssh config, then system-wide config. */
+  parse_ssh_config_file(&options, string(home_dir) + "/.ssh/config");
+  parse_ssh_config_file(&options, "/etc/ssh/ssh_config");
+  cout << "connecting to " << options.host << endl;
```

Comment:
Change to LOG(INFO).  If we want to print to cout, that's fine but let's also add a LOG(INFO) line so it gets logged to file as well.